test/integration/helpers/model-expect.ts

```diff
@@ -167,6 +166,18 @@ type ExpectedModelElement = {
   extraCssClasses?: string[];
 };
 
+export interface ExpectedFill {
+  color?: string;
+  opacity?: Opacity;
+}
```

Comment:
**thought:** needed as in the Fill type, the color can be an object and here, the checks expect it to be a string.

Reply:
If you sax my first implementation for the integration tests, I used the same implementation for the update style API.
But, the calculation of what we really test was hidden from the reading of the tests.

https://github.com/process-analytics/bpmn-visualization-js/pull/2760/commits/bbc9b1e062b70aad2a1877004962f87d1fb124bf

As the direction is not the same value between the API and mxgraph, I needed to change it.